docs/resources/mws_log_delivery.md

```diff
@@ -0,0 +1,59 @@
+# databricks_mws_log_delivery Resource
+
+-> **Note** This resource has evolving API, which may change in future versions of provider.
+
+This resource configures delivery of various log types from databricks workspaces. Backend does not support removal of log delivery configuration, so provider handles it through simply disabling the config.
+
+## Example Usage
+
+Configuring usage logs:
+
+```hcl
+resource "databricks_mws_log_delivery" "usage_logs" {
+    account_id   = var.account_id
+    credentials_id = databricks_mws_credentials.log_writer.credentials_id
+    storage_configuration_id = databricks_mws_storage_configurations.log_bucket.storage_configuration_id
+    config_name = "Usage Logs"
+    log_type = "BILLABLE_USAGE"
+    output_format = "CSV"
+}
+```
+
+Configuring audit logs:
+
+```hcl
+resource "databricks_mws_log_delivery" "audit_logs" {
+    account_id   = var.account_id
+    credentials_id = databricks_mws_credentials.log_writer.credentials_id
+    storage_configuration_id = databricks_mws_storage_configurations.log_bucket.storage_configuration_id
+    config_name = "Audit Logs"
+    log_type = "AUDIT"
```

Comment:
this needs to be "AUDIT_LOGS", and the only supported output format is JSON. Similarly, CSV is the only format supported for usage log delivery now.